### DISCUSSION TITLE
Email body html contents

### DISCUSSION THREAD

**Discussion by ihabmzaky** [Budibase/budibase#6665] [Help] [answered]
I am fairly new to Budibase but managed to trigger an email with fields so I am able to pass along the intended address of the recipient.  But I am NOT able to edit/add content to the body of the email through handlebars nor JavaScript.  Please share an example or a tutorial.  Whatever I try, the email body is blank with budibase link at top and copryrigth at bottom.

**Comment by joebudi** [comment, marked as answer]
Hey, this may help:
https://docs.budibase.com/docs/email#templates

If you are using the email automation, it's probably best to just use plain text in the email body.

**Reply by ihabmzaky** [reply]
Ok, I think I understand the email templates thanks.
Also how can I pass a user email from the Users DB to be a recipient in the email automation based on a form drop down list?  I am trying to send out an email notification to a certain user telling him that he has been assigned a new request.
